script/Arcueid/Day 7/07_00_ARC07_3G.txt

```diff
@@ -601,8 +601,8 @@ You shouldn't strain yourself to make out things you can't see.
 -- こんな希少な魔眼持ちなのに平和主義者とか、おいしい獲物にも程があるし！　よく今まで生きてこられたわね、ひょいってつまんで食べられるわよ、普通！」
 To think that you're a pacifist despite being in possession of such rare Mystic Eyes!
  Do you have any idea how juicy of a target this makes you?
- You've honestly done well to make it this far,
- normally someone would've snatched you up and gobbled you down by now!"
+ You've honestly done well to make it this far.
+ Normally, someone would've snatched you up and gobbled you down by now!"
```

Comment:
isn't the expression "gobble you up"?

Reply:
Yeah, I think I had made it `down` here to make an arc out of snatch **up** / gobble **down**. Can change it to just 
> would've snatched you and gobbled you up by now!

or
> would've plucked you out and gobbled you up by now!

if that's better